storage/store.go

```diff
@@ -917,22 +827,6 @@ func (s *Store) BootstrapRange(initialValues []proto.KeyValue) error {
 	if err := engine.MVCCPutProto(batch, ms, meta1Key, now, nil, desc); err != nil {
 		return err
 	}
-	// Zone config.
-	zoneConfig := &config.ZoneConfig{
-		ReplicaAttrs: []proto.Attributes{
-			{},
-			{},
-			{},
-		},
-		RangeMinBytes: 1048576,
-		RangeMaxBytes: 67108864,
-		GC: &config.GCPolicy{
-			TTLSeconds: 24 * 60 * 60, // 1 day
-		},
-	}
-	if err := engine.MVCCPutProto(batch, ms, keys.ConfigZonePrefix, now, nil, zoneConfig); err != nil {
```

Comment:
For some reason, removing this makes `storage/gc_queue_test.go:TestGCQueueShouldQueue` fail.